shared/studio/tabs/dataview/dataInspector.tsx

```diff
@@ -1021,6 +1026,10 @@ export const MobileDataInspector = ({rowData}: MobileDataInspectorProps) => {
     state.gridRef?.resetAfterRowIndex(rowData.dataRowIndex);
   };
 
+  useEffect(() => {
+    dataviewState.setShowSubtypeFields(true);
+  }, []);
```

Comment:
I'm not sure this is the best way to to this, since this also triggers the underlying table to reload. Currently we filter the subtypes in the function that loads the fields, but instead I think we should move this filtering into a computed, so this component could still access the full set of fields?